### DISCUSSION TITLE
Selling 30BSQ is too expensive. A partial solution is bonded trading.

### DISCUSSION THREAD

**Discussion by MwithM** [bisq-network/bisq#5301] [General]
A couple days ago I took an offer to sell BSQ. Maker was a new user, as expected, trying to use BSQ. I had to:
Set 0.006 BTC min security deposit (400%).
I ftrading fees are paid in BSQ I have to pay 31.8% mining fees at 52sat/vbyte, or 27.03% mining fee at 58 sa/vbyte. Cost is higher paying in BSQ because it doesn't have segwit yet.
I spent 8075 sats (4USD, 6% of the trade) to send the BSQ with a 25 sats/vbyte mining bid.

In total, taking that offer cost me more than a 30% of the trade amount and maker more than 10%.

30BSQ (about 60USD) to spend in trading fees is a sufficient amount to trade during more than a month for most of the traders. Buying higher amounts should be considered an investment in Bisq and not to pay trading fees, unless this trader moves a very high volume.

That's why we need to place a parallel market using a [chat](https://bisq.wiki/Keybase-Buy_Bitcoin) and bonds. I have just set up a [bond for reputation](https://mempool.emzy.de/es/bisq/tx/b9830a33c3c17c8db1de1674c8388e79c2129c4ca2312bcc0772b1d03fdac037), locking 150BSQ for 90 days. How to make this available, easy to understand and secure for buyers and sellers are still into discussion, but I have no doubt this is necessary for Bisq.

**Comment by ripcurlx** [comment]
This will improve as soon as @sqrrm is finalizing the atomic swap between BSQ<>BTC.

**Reply by MwithM** [reply]
That's why I wanted to emphasize that this is a partial solution. Maybe Bisq won't be able to scale to trade coffees, but it should try to be efficient for trades around 100USD, which considerable amount at poor countries.

**Reply by huey735** [reply]
Oh, that's great @ripcurlx . @sqrrm do you have a link to the scheme to be used?

**Reply by sqrrm** [reply]
I'm working on it but it's a big change. So far only in a local branch.
